class.js

```diff
@@ -1,20 +1,23 @@
-class Foo
-{
-color='negro';peso   =    34;
-  ver()
-  {
-    console.info(`Color: ${this.color}. Peso: ${this.peso}`
-    );
-// foo = 123;
-/*
-// Prueba
-test() {
-  console.info('Test');
+class Foo {
+  color = 'negro';
+  peso = 34;
+  ver() {
+    console.info(`Color: ${this.color}. Peso: ${this.peso}`);
+  /*
+  // Prueba
+  test() {
+    console.info('Test');
+  }
+  */
```

Comment:
Este bloque se encuentra dentro del método `ver()`.
```suggestion
    /*
    // Prueba
    test() {
      console.info('Test');
    }
    */
```